flang/runtime/pseudo-unit.cpp

```diff
@@ -0,0 +1,170 @@
+//===-- runtime/pseudo-unit.cpp -------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// Implemenation of ExternalFileUnit and PseudoOpenFile for
+// RT_USE_PSEUDO_FILE_UNIT=1.
+//
+//===----------------------------------------------------------------------===//
+
+#include "io-error.h"
+#include "tools.h"
+#include "unit.h"
+#include <cstdio>
+
+#if defined(RT_USE_PSEUDO_FILE_UNIT)
+
+namespace Fortran::runtime::io {
+
+void FlushOutputOnCrash(const Terminator &terminator) {}
```

Comment:
Some C++ compilers will complain about unused formal arguments; I suggest removing their names, here and below.